.github/workflows/dockerhub.yml

```
@@ -0,0 +1,25 @@
+name: Push to DockerHub
+
+on:
+  push:
+    branches: [master]
+
+jobs:
+  dockerhub:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - uses: satackey/action-docker-layer-caching@v0.0.5
```

```suggestion
      - uses: satackey/action-docker-layer-caching@v0.0.8
```

This suggestion updates it to the latest version, but should we pin this to some hash instead? I see that this is a relatively obscure action. What's our general policy on pinning at the moment? Only pin when there are secrets involved?

---

We should definitely pin. This is an oversight on my part.

---

Okay - I'll update my suggested change then.